src/Endatix.WebHost/Program.cs

```diff
@@ -0,0 +1,39 @@
+using Endatix.Setup;
+using Endatix.Api.Infrastructure;
+using FastEndpoints;
+using FastEndpoints.Swagger;
+using Serilog;
+
+try
+{
+    WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
+
+    var endatixBuilder = builder.UseEndatix()
+                                .UseDefaultSetup()
+                                .UseApiEndpoints();
+
+    // TODO: expose custom plugin logic for plugin custom domain event handlers 

```

Comment:
Most of the TODOs in the code just stay forever. I think creating a task in the backlog has better chance to get the change done.
Of course, if this is something planned as a part of another immediate task (like a reminder), it could be helpful.

Reply:
This is important to be done soon as the App.endatix.com depends on the fix. I have created this task, so we can plan addressing this TODO - https://github.com/endatix/endatix/issues/68. I have added a note to remove the TODO as part of the scope